test/functional/ios/safari/windows-frame-specs.js

```diff
@@ -50,9 +50,12 @@ describe('safari - windows and frames (' + env.DEVICE + ') @skip-ios6"', functio
         .elementById('only_on_page_2')
         .nodeify(done);
     });
-    it("should be able to open js popup windows with safariAllowPopups set to true", function (done) {
+
+    // broken on real devices, see https://github.com/appium/appium/issues/5167
+    it("should be able to open js popup windows with safariAllowPopups set to true @skip-real-device", function (done) {
```

Comment:
so "safariAllowPopups" cap doesn't do anything on real devices of course. Caps.md already says 'sim only' for a lot of these.

The github issue is actually a separate bug which causes the test to fail anyways....